tools/lint-hotkeys

```diff
@@ -39,8 +39,8 @@ def lint_hotkeys_file() -> None:
     existing OUTPUT_FILE
     """
     hotkeys_file_string = get_hotkeys_file_string()
+    error_flag = 0 if is_every_key_category_valid() else 1
```

Comment:
it's rather independent of this change, but it's quite strange to me to use `0` and `1` instead of a boolean here. I know `sys.exit` wants to have an integer, but it should in my opinion only be converted to it at the very last moment.